L/libdrm/build_tarballs.jl

```diff
@@ -3,18 +3,19 @@
 using BinaryBuilder, Pkg
 
 name = "libdrm"
-version = v"2.4.106"
+version = v"2.4.110"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://dri.freedesktop.org/libdrm/libdrm-2.4.106.tar.xz", "92d8ac54429b171e087e61c2894dc5399fe6a549b1fbba09fa6a3cb9d4e57bd4"),
+    ArchiveSource("https://dri.freedesktop.org/libdrm/libdrm-$version.tar.xz", "eecee4c4b47ed6d6ce1a9be3d6d92102548ea35e442282216d47d05293cf9737"),
     DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
 cd libdrm-*
+atomic_patch -p1 ../no_stress.patch
```

Comment:
There must be an easier way to skip this, but I wasn't able to figure out the setup and this seems to work locally for me at least.